Library/Homebrew/formulary.rb

```diff
@@ -33,6 +33,19 @@ def self.formula_class_get(path)
     cache.fetch(path)
   end
 
+  def self.clear_cache
+    cache.each do |k, m|
```

Comment:
```suggestion
    cache.each do |klass, module|
```

assuming that's correct?

Reply:
k for `key` since it's either a pathname or a symbol - it's not consistent. But yes, I'll change these.

Reply:
Ta!